datasets/allen-cell-imaging-collections.yaml

```diff
@@ -8,6 +8,7 @@ Description: |
   4) Machine learning predictions and labels of the data listed above
   5) Models trained on the previously listed data
   6) Additional supporting non-image data related to the above listed data types (e.g., gene expression data, whole genome sequencing data, features derived from the images or model predictions, metadata)
+  7) Simulation, analysis, and visualization data of cell structures, cells, and cell populations
```

Comment:
Could the full line be: Simulation, analysis, and visualization data of in silico cell structures, cells, and cell populations